scripts/apps/tansa/index.ts

```diff
@@ -4,12 +4,15 @@
  */
 
 /* jshint ignore:start */
-document.write(`
+
+// tansa server base urls e.g 'http://localhost:8080/tansaclient/'
+var baseUrl = `${location.protocol}//${location.host}/tansaclient/`;
+var tansaScript =
+`
     <script type="text/javascript">
         var tansa = {
             settings: {
-                // tansa server base urls e.g 'http://localhost:8080/tansaclient/'
-                baseUrl: '${location.protocol}//${location.host}/tansaclient/',
+                baseUrl: '<baseUrl>',
```

Comment:
How is this related to `nggettext_extract`? I see no `gettext` tags here.

Reply:
nggettext_extract was failing to parse the file:

ivp:superdesk-client-core ivp$ grunt nggettext_extract
Running "nggettext_extract:pot" (nggettext_extract) task
Error parsing scripts/apps/tansa/index.ts at line 5 column 26

Reply:
Did it fail in the command line when doing the extraction? Will it be fine now for all other files?

Reply:
Only this file had problems on nggettext_extract parsing

>ivp:superdesk-client-core ivp$ grunt nggettext_extract
Running "nggettext_extract:pot" (nggettext_extract) task
>
>Done.